sourcefiles/itemrando.py

```diff
@@ -56,6 +60,37 @@ def write_item_prices_to_config(settings: rset.Settings,
         config.item_db[item].price = price
 
 
+def update_snail_stop_price(ct_rom: ctrom.CTRom):
+    # Modify Snail Stop Item Price
+    OID = 0x09
+
+    script = ct_rom.script_manager.get_script(ctenums.LocID.SNAIL_STOP)
+    pos = script.get_function_start(OID, FID.ACTIVATE)
+
+    # Price range 
+    #   - Lower: 7500 (which is about %75 original)
+    #   - Upper: 17500 (which is about %175 original)
+    price = random.randint(7500, 17500)//100*100 # Round to nearest 100
+
+    # Update Text in Decision Box
+    pos, _ = script.find_command([0xC0], pos)
+    str_id = script.data[pos+1]
+    script.strings[str_id] = ctstrings.CTString.from_str(
+        rf"I might let some go for, say, {price}.{{line break}}"
```

Comment:
Should probably be {price}G to match the original.

Reply:
Fixed